src/roadmap/data/app_streams.py

```diff
@@ -124,6 +127,16 @@ def set_display_name(self):
 
         return self
 
+    @model_validator(mode="after")
+    def update_support_status(self):
+        """Validator for setting status."""
+        today = date.today()
+        self.support_status = _calculate_support_status(
```

Comment:
should this be made public instead?

Reply:
Potentially. It's shared across multiple classes and is internal to the implementation. We _could_ change how this works in the future, making it best to keep it private for now.

I tend to air on the side of making things private unless they _really_ need to be public.